### PR TITLE
Update Complete_Rec.mdx (Make code consistent)

### PR DESCRIPTION
**Pull request by tachithanhdanh**

### CODE REVIEW

File: content/2_Bronze/Complete_Rec.mdx
Comment (Farrius) [inline content/2_Bronze/Complete_Rec.mdx:610]: Fix the space between ] and ++

File: content/2_Bronze/Complete_Rec.mdx
Comment (Farrius) [inline content/2_Bronze/Complete_Rec.mdx:1014]: Code looks good but in this case I think the code with templates is fine, and there is no need to change it. Nevertheless wait for another review.

**Review by Farrius — CHANGES_REQUESTED** [review]

**Review by darren-yao — APPROVED** [review]

**Review by darren-yao — APPROVED** [review]

**Review by darren-yao — APPROVED** [review]

**Pull request merged**
